joomla/joomla-cms
# Improve UX for login / logout redirect options

Pull Request for Issue #11176  .

### Summary of Changes
- Removed return false if Menu Item is selected and URL is filled
- Added clear the field to avoid conflicts

Test instructions:

Before the Patch:
1. Create or open a Login Menu Item
2. chose the Internal URL redirect type in login or logout (doesn´t matter)
3. type in for example just index.php
4. Save
5. change the redirect type to Menu Item and select any Menu item
6. Click on save

Actual result: You get an warning message "Only one of the login redirect fields should have a value." the new selection is not saved. -> This is no good UX and might confuse a lot of users

After the patch:
1. Create or open a Login Menu Item
2. chose the Internal URL redirect type in login or logout (test both)
3. type in for example just index.php
4. Save
5. change the redirect type to Menu Item and select any Menu item
6. Click on save.

The Internal URL redirect will be cleared, if you select an Menu Item Type and save.
The Menu Item Type will be cleared, if you select Internal URL and save.
